src/OrchardCore/OrchardCore.ContentManagement.Abstractions/ContentItemExtensions.cs

```diff
@@ -95,6 +95,16 @@ public static ContentItem Merge(this ContentItem contentItem, object properties,
                 contentItem.DisplayText = props[nameof(contentItem.DisplayText)].ToString();
             }
 
+            if (props.ContainsKey(nameof(contentItem.Owner))) {
+                contentItem.Owner = props[nameof(contentItem.Owner)].ToString();
+                contentItem.Author = props[nameof(contentItem.Owner)].ToString();
+            }
+
+            if (props.ContainsKey(nameof(contentItem.Author)))
```

Comment:
What about all other properties

Reply:
I took care of the properties that we're not assigned when looking in the ContentItemIndex table. If we add other properties over time that are not affected when doing a .Merge() then we will require to add it here unless we find a better way. I think it's kind of normal that we should assign these specific properties as a .Merge() or .Apply() never affects these values. One will change the .Data and the other one will change the .Content if I remember well. So, when we pass a ContentItem through the ApiController we require to Merge the received ContentItem with the .NewAsync() generated one for example. And in this particular use case even if we had assigned an Owner to the received ContentItem it would never be affected when doing .Merge or .Apply.